packages/geofire/src/GeoQuery.ts

```diff
@@ -6,22 +6,21 @@ import {
 import {
   decodeGeoFireObject, geoFireGetKey } from './databaseUtils';
 import * as GeoFireTypes from './GeoFireTypes';
-import * as DatabaseTypes from '@firebase/database-types';
+import { 
+  DataSnapshot, orderByChild, DatabaseReference, query, startAt, endAt, off, child, get, Query, onChildAdded, onChildRemoved,
+  onChildChanged, onValue, Unsubscribe 
+} from 'firebase/database';
 
 export interface QueryCriteria {
   center?: Geopoint;
   radius?: number;
 }
-export type QueryStateCallback = (
```

Comment:
It's actually cool to see how this is now replaced by a built-in type of the SDK. 👍